javascripts/card.js

```diff
@@ -0,0 +1,49 @@
+var Carlot = (function(car) {
+  var editor;
+
+	car.cardFocus = function (element) {
+		textInput.removeAttribute("disabled", "true");
+		textInput.focus("true");
+		checkPurchased.removeAttribute("disabled", "true");
+		element.classList.add("card-focus");
+		// var edit = element.querySelector("");
+		// edit.removeAttribute("style");
+		// edit.classList.add("cardFocus");
+		labelPurchased.classList.remove("disabled");
+		editor = element;
+		// currentList = edit;
+		Carlot.cardEdit();
+	};
+
+	car.cardEdit = function () {
+		var inventory = Carlot.getInventory();
+		currentInventory = inventory.find(function(car) {
```

Comment:
Here you're using ```currentInventory``` before it's defined, and later you use this same variable in the ```cardFocusOff``` function.  For this, the variable needs to be declared at the root scope of the IIFE, so it should be declared up where you're declaring ```editor```.

Not defining it explicitly causes JavaScript to declare it for you as a truly global variable, which is undesirable.